src/js/pages/LoginPage.js

```diff
@@ -76,73 +76,171 @@ export default class LoginPage extends Component {
-            <Button onPress={this.handleSubmit} title={ i18n.t('login.loginButton') } { ...testProperties(i18n.t('login.loginButton')) } />
-
-            {errorMessage}
-
-            <Text style={styles.login_info}>{ i18n.t('login.loginText') }</Text>
+      <ScrollView
+        contentContainerStyle={ styles.scrollContainer }
+        keyboardShouldPersistTaps="handled"
+        { ...testProperties(i18n.t('login.screen')) }
+      >
+        <View style={ styles.wrapper }>
+          <View style={ styles.loginContainer }>
+            <Image
+              resizeMode="contain"
+              source={ require('../../img/swag.labs.logo.png') }
+              style={ styles.swagLogo }
+            />
+            <Input
+              containerStyle={ styles.login_input }
+              inputContainerStyle={ styles.inputContainerStyle }
+              placeholder={ i18n.t('login.username') }
+              value={ this.state.username }
+              onChangeText={ this.handleUserChange }
+              shake={ true }
+              autoCapitalize="none"
+              autoCorrect={ false }
+              { ...testProperties(i18n.t('login.username')) }
+            />
+            <Input
+              containerStyle={ styles.login_input }
+              inputContainerStyle={ styles.inputContainerStyle }
+              placeholder={ i18n.t('login.password') }
+              value={ this.state.password }
+              onChangeText={ this.handlePassChange }
+              shake={ true }
+              secureTextEntry={ true }
+              { ...testProperties(i18n.t('login.password')) }
+            />
+            { errorMessage }
+            <Button
+              buttonStyle={ styles.buttonStyle }
+              containerStyle={ styles.buttonContainerStyle }
+              titleStyle={ styles.buttonTitleStyle }
+              onPress={ this.handleSubmit }
+              title={ i18n.t('login.loginButton') }
+              { ...testProperties(i18n.t('login.loginButton')) }
+            />
+            <Image
+              source={ require('../../img/login.bot.png') }
+              style={ styles.loginBot }
+              resizeMode="contain"
+            />
+          </View>
+          <View style={ styles.loginInfoContainer }>
+            <Text style={ styles.login_info }>
+              { this.parseNormalBoldText(i18n.t('login.loginText.usernames')) }
+            </Text>
+            <Divider style={ styles.divider }/>
+            <Text style={ styles.login_info }>
+              { this.parseNormalBoldText(i18n.t('login.loginText.password')) }
+            </Text>
           </View>
-        </ScrollView>
-      </ThemeProvider>
+        </View>
+      </ScrollView>
     );
   }
 }
 
 const styles = StyleSheet.create({
   scrollContainer: {
-    flex: 1,
+    backgroundColor: colors.white,
+    paddingTop: IS_IOS ? (IS_IPHONEX ? 44 : 20) : (StatusBar.currentHeight || 0),
```

Comment:
Isn't this duplicated logic from StatusBar.STATUS_BAR_HEIGHT? Should we be using that constant here instead?